cvmfs/util/algorithm.h

```diff
@@ -9,15 +9,22 @@
 
 #include <algorithm>
 #include <cassert>
+#include <cmath>
```

Comment:
This and the following includes should not be needed, at least in the header

Reply:
Removed them